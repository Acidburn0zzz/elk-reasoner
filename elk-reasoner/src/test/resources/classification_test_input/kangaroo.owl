Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(
SubClassOf(<Anonymous-18#Beast> ObjectIntersectionOf(<Anonymous-18#Irrational> <Anonymous-18#Animal>))
SubClassOf(<Anonymous-18#Human> ObjectIntersectionOf(<Anonymous-18#Rational> <Anonymous-18#Animal>))
DisjointClasses(<Anonymous-18#Irrational> <Anonymous-18#Rational>)
SubClassOf(<Anonymous-18#Kangaroo> <Anonymous-18#Beast>)
SubClassOf(<Anonymous-18#KangarooInfant> ObjectIntersectionOf(ObjectSomeValuesFrom(<Anonymous-18#lives-in> <Anonymous-18#Pouch>) <Anonymous-18#Kangaroo>))
EquivalentClasses(<Anonymous-18#MaternityKangaroo> ObjectIntersectionOf(<Anonymous-18#Kangaroo> ObjectSomeValuesFrom(<Anonymous-18#has-body-part> <Anonymous-18#Pouch>) ObjectSomeValuesFrom(<Anonymous-18#has-child> <Anonymous-18#KangarooInfant>)))
EquivalentClasses(<Anonymous-18#Parent> ObjectSomeValuesFrom(<Anonymous-18#has-child> owl:Thing))
DisjointClasses(<Anonymous-18#Rational> <Anonymous-18#Irrational>)
Declaration(ObjectProperty(<Anonymous-18#has-child>))
ObjectPropertyDomain(<Anonymous-18#has-child> <Anonymous-18#Human>)
ObjectPropertyRange(<Anonymous-18#has-child> <Anonymous-18#Human>)
)
