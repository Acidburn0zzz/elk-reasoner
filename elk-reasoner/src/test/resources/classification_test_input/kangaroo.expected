Ontology(
Declaration(Class(<test:KangarooInfant>))
Declaration(Class(<test:Kangaroo>))
Declaration(Class(<test:Human>))
Declaration(Class(<test:Beast>))
Declaration(Class(<test:Rational>))
Declaration(Class(<test:Pouch>))
Declaration(Class(<test:Irrational>))
Declaration(Class(<test:MaternityKangaroo>))
Declaration(Class(<test:Animal>))
Declaration(Class(<test:Parent>))
EquivalentClasses(<http://www.w3.org/2002/07/owl#Nothing> <test:MaternityKangaroo>)
SubClassOf(<test:Beast> <test:Animal>)
SubClassOf(<test:Human> <test:Animal>)
SubClassOf(<test:Kangaroo> <test:Beast>)
SubClassOf(<test:Parent> <test:Human>)
SubClassOf(<test:Beast> <test:Irrational>)
SubClassOf(<test:KangarooInfant> <test:Kangaroo>)
SubClassOf(<test:Human> <test:Rational>)
)